### PR TITLE
Update translations for 1.0.0

### PR DESCRIPTION
**Pull request by rmol**
## Status

Ready for review

## Description of Changes

Updated translations for 1.0.0.

## Testing

- Check out this branch.
- Run the page layout tests: `make test TESTFILES=tests/pageslayout`
- Check some of the screenshots under `securedrop/tests/pageslayout/screenshots/`
- Run the development server and check the strings changed for 1.0.0 by visiting with a couple of languages and verifying that the strings are in fact translated. The source start page has several, e.g. "First submission" and "Return visit".

## Deployment

No deployment concerns.

### If you made non-trivial code changes:

- [x] I have written a test plan and validated it for this PR

### CODE REVIEW

**Comment by conorsch** [comment]
[Most recent build](https://circleci.com/gh/freedomofpress/securedrop/34051) shows CI failure as described in #4818. Since #4819 landed on develop, a rebase on this branch should resolve, getting the translation tests passing reliably again.

File: securedrop/translations/ro/LC_MESSAGES/messages.po [hunk 1]
```diff
@@ -983,6 +983,16 @@ msgstr ""
 msgid "Forgot your codename?"
 msgstr "Ți-ai uitat numele de cod?"
 
+#: source_templates/footer.html:7
```
Comment (emkll) [inline securedrop/translations/ro/LC_MESSAGES/messages.po]: Let's make sure we test the footer behavior with `ro` and ensure the english-language messages are properly displayed for these strings.

Comment (rmol) [reply securedrop/translations/ro/LC_MESSAGES/messages.po]: They are.

romanian

**Comment by emkll** [comment]
All translation tests passed in https://circleci.com/gh/freedomofpress/securedrop/34351 , merging!

**Pull request merged**